src/mame/konami/thunderx.cpp

```diff
@@ -441,30 +438,28 @@ void thunderx_state::calculate_collisions()
 	// hit mask is 40 to set bit on object 0 and object 1
 	// hit mask is 20 to set bit on object 1 only
 
-	Y0 = m_pmcram[0];
-	Y0 = (Y0 << 8) + m_pmcram[1];
-	Y0 = (Y0 - 15) / 5;
-	Y1 = (m_pmcram[2] - 15) / 5;
+	e0 = m_pmcram[0];
+	e0 = (e0 << 8) + m_pmcram[1];
+	e1 = m_pmcram[2];
```

Comment:
If you’re changing this anyway, you may as well declare `e0` and `e1` at initialisation, initialise `e0` in a single statement, and make them both `const`.